Hey @behrmann With "incremental=no", it is showing 7 failed tests and i think "default-initrd.cache does not exist" in logs means the cache is getting removed.

Here is [link to the raw log output ](https://productionresultssa0.blob.core.windows.net/actions-results/0de1e242-a9e0-4889-8a0e-ad887ab89044/workflow-job-run-cdb4750a-162a-53f3-97ef-ad984165a2f9/logs/job/job-logs.txt?rsct=text%2Fplain&se=2026-07-21T19%3A05%3A51Z&sig=O1vU8GiwjsnLbslb58xzWslY28zkUnnFDVz5WFHSolk%3D&ske=2026-07-21T21%3A24%3A00Z&skoid=ca7593d4-ee42-46cd-af88-8b886a2f84eb&sks=b&skt=2026-07-21T17%3A24%3A00Z&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skv=2025-11-05&sp=r&spr=https&sr=b&st=2026-07-21T18%3A55%3A46Z&sv=2025-11-05)of one of the 7 failing CI checks, specifically these lines 

```
2026-07-20T19:50:51.6024838Z ‣ /home/runner/work/mkosi/mkosi/mkosi.cache/fedora~rawhide~x86-64~default-initrd.cache does not exist, not reusing cached images
2026-07-20T19:50:51.6026197Z ‣ Strict incremental mode is enabled and cache for image default-initrd is out-of-date
2026-07-20T19:50:51.6026874Z ‣ (Build once with '-i yes' to update the image cache)
2026-07-20T19:50:51.6246787Z Traceback (most recent call last):
2026-07-20T19:50:51.6253892Z   File "/home/runner/work/mkosi/mkosi/mkosi/run.py", line 104, in uncaught_exception_handler
2026-07-20T19:50:51.6254536Z     yield
2026-07-20T19:50:51.6254913Z   File "/usr/lib64/python3.15/contextlib.py", line 116, in inner
2026-07-20T19:50:51.6255582Z     return func(*args, **kwds)
2026-07-20T19:50:51.6256086Z   File "/home/runner/work/mkosi/mkosi/mkosi/__main__.py", line 44, in main
2026-07-20T19:50:51.6256675Z     run_verb(args, tools, images, resources=resources)
2026-07-20T19:50:51.6257204Z     ~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-07-20T19:50:51.6257764Z   File "/home/runner/work/mkosi/mkosi/mkosi/__init__.py", line 5251, in run_verb
2026-07-20T19:50:51.6258302Z     die(
2026-07-20T19:50:51.6258540Z     ~~~^
2026-07-20T19:50:51.6259100Z         f"Strict incremental mode is enabled and cache for image {c.image} is out-of-date",
2026-07-20T19:50:51.6259767Z         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-07-20T19:50:51.6260305Z         hint="Build once with '-i yes' to update the image cache",
2026-07-20T19:50:51.6260790Z         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2026-07-20T19:50:51.6261164Z     )
2026-07-20T19:50:51.6261399Z     ^
2026-07-20T19:50:51.6261767Z   File "/home/runner/work/mkosi/mkosi/mkosi/log.py", line 35, in die
2026-07-20T19:50:51.6262608Z     sys.exit(1)
2026-07-20T19:50:51.6262877Z     ~~~~~~~~^^^
2026-07-20T19:50:51.6263138Z SystemExit: 1
```

I can try "incremental=yes" also but wanted to show you these first.

--- a/tests/test_boot.py
+++ b/tests/test_boot.py
@@ -85,7 +85,7 @@ def test_bootloader(config: ImageConfig, bootloader: Bootloader) -> None:
                 "--format=disk",
                 "--bootloader",
                 str(bootloader),
-                *(["--incremental=no"] if bootloader == Bootloader.uki_signed else []),
+                *(["--incremental=yes"] if bootloader == Bootloader.uki_signed else []),
             ]
         )
         image.vm(["--firmware", str(firmware)])

